src/server/twitterSearch.js

```diff
@@ -261,6 +264,41 @@ export const searchAndSaveFromTwitter = (query, count = 300) => {
   }
 };
 
+const sweepTwitterAndConcat = (query, count, existingStatuses = [], lowestId = null) => {
+  let extendedQuery = query;
+  if (lowestId !== null) {
+    extendedQuery += ` max_id:${lowestId}`;
+  };
+
+  return newPromiseChain()
+    .then(() => potentiallySearchTwitter(extendedQuery, count))
+    .then((twitStatuses) => {
+      const countLeft = count - 100;
```

Comment:
how comes you take away a static 100 instead of:
`const countLeft = count - added` the added that's below